documentation/ranking.html

```diff
@@ -509,29 +509,6 @@ <h2 id="using-query-variables">Using query variables</h2>
 
 
 
-<h2 id="literal-boosting">Literal boosting</h2>
-<p>
-By default, Vespa does <a href="linguistics.html#stemming">stemming</a> and
-<a href="linguistics.html#normalization">normalization</a>
-of the words in the indexes and queries to increase recall.
-Some times it helps relevance to write a ranking expression which gives a higher
-score to matches where the <em>literal</em> form used in the query matches the
-literal form used in the document.
-Add <code>rank:literal</code> to the field(s) to enable.
-Then write a ranking function which accesses features for a field name
-as the original field with <code>_literal</code> appended.
-Example field and ranking expression:
-<pre>
-field title type string {
-    indexing: index
-    rank:     literal
-}
-</pre><pre>
-0.9*fieldMatch(title) + 0.1*fieldMatch(title_literal)</pre>
-</p>
-
-
```

Comment:
so https://docs.vespa.ai/documentation/reference/search-definitions-reference.html#rank should be updated too? rank:literal is gone ?  no mention at https://docs.vespa.ai/documentation/vespa7-release-notes.html. confused

Reply:
I think since Vespa 4 we have not generated _literal fields.

Reply:
Let met get rid of the rank:literal as well